frontend/src/pages/ShootItemTerminal.vue

```diff
@@ -15,737 +15,225 @@ limitations under the License.
-          </v-btn>
-          {{terminalSession.detailedConnectionStateText || connectionStateText}}
-        </v-tooltip>
-        <v-list>
-          <v-list-tile-action v-if="terminalSession.connectionState === TerminalSession.DISCONNECTED" >
-            <v-btn small slot="activator" flat class="ml-2 mr-2 cyan--text text--darken-2" @click="retry()">
-              <v-icon left>mdi-reload</v-icon>
-              Reconnect
-            </v-btn>
-          </v-list-tile-action>
-          <v-list-tile-content v-else class="ml-2 mr-2">
-            {{terminalSession.detailedConnectionStateText || connectionStateText}}
-          </v-list-tile-content>
-        </v-list>
-      </v-menu>
-
-      <v-tooltip v-if="imageShortText" top>
-        <v-btn small flat slot="activator" @click="configure('imageBtn')" :loading="loading.imageBtn" class="text-none grey--text text--lighten-1 systemBarButton">
-          <v-icon class="mr-2">mdi-layers-triple-outline</v-icon>
-          <span>{{imageShortText}}</span>
-        </v-btn>
-        Image: {{terminalSession.image}}
-      </v-tooltip>
-
-      <v-tooltip v-if="privilegedMode !== undefined && target === 'shoot'" top>
-        <v-btn small flat slot="activator" @click="configure('secContextBtn')" :loading="loading.secContextBtn" class="text-none grey--text text--lighten-1 systemBarButton">
-          <v-icon class="mr-2">mdi-shield-account</v-icon>
-          <span>{{privilegedModeText}}</span>
-        </v-btn>
-        <strong>Privileged:</strong> {{terminalSession.privileged}}<br/>
-        <strong>Host PID:</strong> {{terminalSession.hostPID}}<br/>
-        <strong>Host Network:</strong> {{terminalSession.hostNetwork}}
-      </v-tooltip>
-
-      <v-tooltip v-if="terminalSession.node && target === 'shoot'" top>
-        <v-btn small flat slot="activator" @click="configure('nodeBtn')" :loading="loading.nodeBtn" class="text-none grey--text text--lighten-1 systemBarButton">
-          <v-icon :size="14" class="mr-2">mdi-server</v-icon>
-          <span>{{terminalSession.node}}</span>
-        </v-btn>
-        Node: {{terminalSession.node}}
-      </v-tooltip>
-
-      <v-spacer></v-spacer>
-
-      <v-tooltip top>
-        <v-btn small flat slot="activator" @click="configure('settingsBtn')" :loading="loading.settingsBtn" class="text-none grey--text text--lighten-1 systemBarButton">
-          <v-icon>mdi-settings</v-icon>
-        </v-btn>
-        Settings
-      </v-tooltip>
-
-      <v-divider vertical></v-divider>
-
-      <v-btn :disabled="!isTerminalSessionCreated" small flat class="text-none grey--text text--lighten-1 systemBarButton" @click="deleteTerminal">
-        <v-icon class="mr-2">mdi-exit-to-app</v-icon>
-        Exit
-      </v-btn>
-    </v-system-bar>
-    <terminal-settings-dialog
-      ref="settings"
-      :target="target"
-    ></terminal-settings-dialog>
-    <confirm-dialog ref="confirmDialog"></confirm-dialog>
-  </v-layout>
+      <template v-slot="{item}">
```

Comment:
remove template if possible

Reply:
does not work, there is the error `Property or method "item" is not defined on the instance but referenced during render. Make sure that this property is reactive, either in the data option, or for class-based components, by initializing the property`